pkg/management/component.go

```diff
@@ -474,8 +474,8 @@ func (c *component) ResetPassword(ctx context.Context, realmName string, userID
 	}
 
 	//store the API call into the DB
-	// the error should be treated
-	_ = c.reportEvent(ctx, "INIT_PASSWORD", database.CtEventRealmName, realmName, database.CtEventUserID, userID)
+	//_ = c.reportEvent(ctx, "INIT_PASSWORD", database.CtEventRealmName, realmName, database.CtEventUserID, userID)
```

Comment:
If not used, you'd better completely remove this block

Reply:
I would prefer to leave it as I don't know if the back office will start using it or not soon

Reply:
ok

Reply:
If we decide to leave the functionality, I would suggest to also leave the reportEvent.

Reply:
done